Description: If in a Unity session, don't allow the user to disable Unity from main view.
Author: Andrew Starr-Bochicchio <a.starr.b@gmail.com>

=== modified file 'ccm/Widgets.py'
diff -Nur -x '*.orig' -x '*~' a/compizconfig/ccsm/ccm/Widgets.py b/compiconfig/ccsm/ccm/Widgets.py
--- a/compizconfig/ccsm/ccm/Widgets.py	2012-02-02 19:05:53.690931000 -0500
+++ b/compizconfig/ccsm/ccm/Widgets.py	2012-02-13 18:42:47.612544975 -0500
@@ -1401,7 +1401,11 @@
         button.set_tooltip_text (plugin.LongDesc)
         button.add (box)
 
-        if plugin.Name != 'core':
+        blacklist_plugins = ['core']
+        if os.getenv('DESKTOP_SESSION') == 'ubuntu':
+            blacklist_plugins.append('unityshell')
+
+        if plugin.Name not in blacklist_plugins:
             enable = gtk.CheckButton ()
             enable.set_tooltip_text(_("Enable %s") % plugin.ShortDesc)
             enable.set_active (plugin.Enabled)
